src/Symfony/Component/Process/ProcessBuilder.php

```diff
@@ -226,6 +226,22 @@ public function setOption($name, $value)
     }
 
     /**
+     * Adds a proc_open options.
+     *
+     * @param array $options Key value pair of options
+     *
+     * @return ProcessBuilder
+     */
+    public function setOptions(array $options = array())
+    {
+        foreach ($options as $name => $value) {
```

Comment:
Shoud be `$this->options = $options;`. I believe symfony differs between `addOptions` vs. `setOptions` in this case.